node/src/client/mod.rs

```diff
@@ -181,6 +203,73 @@ impl<N: Network, C: ConsensusStorage<N>> Client<N, C> {
         }));
     }
 
+    /// Initializes transaction verification.
+    fn initialize_transaction_verification(&self) {
+        // Start the transaction verification loop.
+        let node = self.clone();
+        self.handles.lock().push(tokio::spawn(async move {
+            loop {
+                // If the Ctrl-C handler registered the signal, stop the node.
+                if node.shutdown.load(Relaxed) {
+                    info!("Shutting down transaction verification");
+                    break;
+                }
+
+                // Sleep briefly to allow transactions to be validated.
+                tokio::time::sleep(std::time::Duration::from_millis(100)).await;
```

Comment:
perhaps this value could be influenced by `queue_len`?

Reply:
What about d2ba98384
Keeping in mind we should keep it simple to review